resources/language/resource.language.fr_fr/strings.po

```diff
@@ -648,19 +648,19 @@ msgstr "Vue pour la recherche"
 
 msgctxt "#30161"
 msgid "View for exported"
-msgstr "Vue pour l'export"
+msgstr "Vue pour le contenu exporté"
 
 msgctxt "#30162"
 msgid "Last used"
-msgstr "Dernière utilisation"
+msgstr "Dernière utilisée"
 
 msgctxt "#30163"
 msgid "Audio description"
 msgstr "Audiodescription"
 
 msgctxt "#30164"
 msgid "Browse contents with audio description."
-msgstr "Parcourir le contenu avec Audiodescription"
+msgstr "Parcourez les programmes disponibles en audiodescription"
```

Comment:
missing final dot

Reply:
Nice catch :) I will fix it in my next commit.